pages/trouw-opties/[slug].tsx

```diff
@@ -32,59 +33,33 @@ import {
 } from "../../src/components";
 import { PageFooterTemplate } from "../../src/components/huwelijksplanner/PageFooterTemplate";
 import { PageHeaderTemplate } from "../../src/components/huwelijksplanner/PageHeaderTemplate";
-import { calendars, CeremonyType } from "../../src/data/huwelijksplanner-state";
-import { Availability } from "../../src/generated";
-import { HuwelijksplannerAPI } from "../../src/openapi";
+import { CalendarEvent } from "../../src/data/huwelijksplanner-state";
+import { AvailabilitycheckService } from "../../src/generated";
 
 export const getServerSideProps = async ({ locale }: { locale: string }) => ({
   props: {
     ...(await serverSideTranslations(locale, ["common", "huwelijksplanner-step-2"])),
   },
 });
 
-const BlogPost: NextPage = () => {
-  const { locale = "nl", push } = useRouter();
+const PlanningFormPage: NextPage = () => {
+  const { locale = "nl", replace } = useRouter();
```

Comment:
Hmm why use `replace` here instead of `push`? Is there a reason we do not want the user to be able to navigate back with the browser back-link?

Reply:
Correct! Currently only router usage on this page is to send the user back in case there is an invalid state (no marriage type selected). I'll be adding push for navigating forward :)